### ISSUE TITLE
Masking support in BatchNorm

### ISSUE THREAD

**Issue by 3lim** [keras-team/keras#18759]
Keras2 supports masking in its BatchNorm implementation. While Keras3 seems to accept a "mask" param, is it not used anywhere. This seems inconsistent.

**Comment by fchollet** [comment]
Indeed, we should add masking support in the BatchNormalization layer. Is this something you'd be able to open a PR for?

**Comment by sachinprasadhs** [comment]
We can close this issue since masking is now supported in `BatchNormalization` here https://keras.io/api/layers/normalization_layers/batch_normalization/
